schema/definitions/0.8.0/examples/surface_faultroom_data.yml

```diff
@@ -0,0 +1,169 @@
+# Example metadata for a depth surface.
+
+$schema: https://main-fmu-schemas-dev.radix.equinor.com/schemas/0.8.0/fmu_results.json
+version: "0.8.0" # including for REP and others to build logic on.
+source: fmu  # always included, and always this value if FMU made it. For quick identification for external consumers.
+
+tracklog:
+  - datetime: 2020-10-28T14:28:02
+    user:
+      id: jriv
+    event: created
+    sysinfo:
+      fmu-dataio:
+        version: 1.2.3
+      komodo: # only added when running in Komodo environment
+        version: 2023.12.05-py38
+  - datetime: 2020-10-28T14:46:14
+    user:
+      id: peesv
+    event: updated
+
+class: surface # the geojson from fault-room represents a surface (triangulated)
+
+fmu: # the fmu-block contains information directly related to the FMU context
+  model:
+    name: ff
+    revision: 21.0.0.dev
+    description:
+      - detailed description
+      - optional
+
+  workflow:
+    reference: rms/structural_model
+
+  case:
+    name: MyCaseName
+    uuid: 8bb56d60-8758-481a-89a4-6bac8561d38e
+    user:
+      id: jriv # $USER from ERT
+    description:
+      - yet other detailed description
+      - optional
+
+  iteration:
+    id: 0 # always an int, will be 0 for e.g. "pred"
+    uuid: 4b939310-34b1-4179-802c-49460bc0f799 # hash of case.uuid + iteration.id
+    name: "iter-0" # /"pred"
+    restart_from: 15ce3b84-766f-4c93-9050-b154861f9100   # fmu.iteration.uuid for another iteration
+
+  realization:
+    id: 33
+    uuid: 29a15b21-ce13-471b-9a4a-0f791552aa51
+    name: "realization-33"
+    parameters: # directly pass parameters.txt. This is potentially a lot of content, only a stub is included here.
+      SENSNAME: faultseal
+      SENSCASE: low
+      RMS_SEED: 1006
+      INIT_FILES:
+        PERM_FLUVCHAN_E1_NORM: 0.748433
+        PERM_FLUVCHAN_E21_NORM: 0.782068
+      KVKH_CHANNEL: 0.6
+      KVKH_US: 0.6
+      FAULT_SEAL_SCALING: 0.1
+      FWL_CENTRAL: 1677
+
+  context:
+    stage: realization
+
+file:
+  # the Faultroom polygons are some kind of triangulated surfaces
+  relative_path: realization-33/iter-0/share/results/maps/topvolantis_faults.geojson # case-relative
```

Comment:
Maybe make filename more similar to actual output (with the short hash, etc)

Reply:
good catch